modules/gdscript/editor/script_templates/CharacterBody3D/godot_character_controller.gd

```diff
@@ -0,0 +1,73 @@
+# meta-description: Godot character controller for gravity games (FPS, TPS, ...)
+
+extends _BASE_
+
+
+@export_group("Kinematic")
```

Comment:
Shouldn't the group be called "character"?

Reply:
I think both can work, I indicated Kinematic to refer to the movement (the function), Character would refer to the node (the targe as the group Camera).

Change into a character is fine with me, let's see if there are other reactions.